cli/azd/pkg/httputil/util.go

```diff
@@ -119,3 +119,19 @@ func RetryAfter(resp *http.Response) time.Duration {
 
 	return 0
 }
+
+// GetIpAddress returns the public IP address of the caller.
+func GetIpAddress() (string, error) {
+	resp, err := http.Get("https://api.ipify.org")
```

Comment:
If we must add this, let's figure out a way to do it using an azure backed service (ideally one we don't run!).  But this feels like the sort of thing that's going to be a footgun